LICENSE.md

```diff
@@ -6,15 +6,19 @@ Version 2.0.3
 0. Definitions
 --------------
 
+**Source Code**: The preferred form of the work for making modifications to it.
+
+**Object code**: Any non-source form of a work. 
```

Comment:
What is the purpose of adding "object code"? What is the difference between object code and source code?